tmNIDS

```diff
@@ -38,6 +41,11 @@ test_useragent () {
 test_badcas () {
   curl -s https://edellroot.badssl.com/ > /dev/null
```

Comment:
@heyibrahimkhan I think we can move this one and the one underneath, no? IIRC, signatures were disabled in ruleset.

Reply:
Yeah. Both disabled rules for edellroot and superfish removed